tools/run_tests/performance/scenario_config.py

```diff
@@ -348,7 +375,7 @@ def scenarios(self):
           'server_type': 'ASYNC_GENERIC_SERVER',
           'security_params': secargs,
           'core_limit': 0,
-          'async_server_threads': 1,
+          'async_server_threads': 0,
```

Comment:
For a single-channel single-depth ping-pong, I think it makes sense to use 1 async_server_thread if this is meaningful for your language.